handlers/refunds.go

```diff
@@ -63,4 +65,11 @@ func HandleCreateRefund(w http.ResponseWriter, req *http.Request) {
 	}
 
 	log.InfoR(req, "Successful POST request for new refund", log.Data{"refund_id": refund.RefundId, "status": http.StatusCreated})
+
+	err = handleRefundMessage(paymentResource.MetaData.ID, refund.RefundId)
+	if err != nil {
+		log.ErrorR(req, fmt.Errorf("error producing kafka message: [%v]", err))
```

Comment:
might be good to differentiate between payment kafka message errors vs refund kafka message errors.